dev/source/docs/mavlink-rcinput.rst

```diff
@@ -0,0 +1,324 @@
+.. _mavlink-rcinput:
+
+==========================
+RC Input (aka Pilot Input)
+==========================
+
+Traditionally the pilot's manual input via an RC transmitter is sent to the autopilot via a separate wireless link but it can alternatively be sent via MAVLink.  This page describes the messages that can be used and gives advice on some potential issues
+
+For reference here are some user focused RC related wiki pages
+
+- :ref:`Joystick/Gamepad setup <copter:common-joystick>`
+- :ref:`RC Input Channel Mapping (aka RCMAP) <copter:common-rcmap>`
+- :ref:`Auxiliary Functions <copter:common-auxiliary-functions>`
+
+The autopilot will ignore the RC input messages if the sender's system id does not match the autopilot's :ref:`SYSID_MYGCS <copter:SYSID_MYGCS>` and :ref:`SYSID_ENFORCE <copter:SYSID_ENFORCE>` = 1
+
+RC input via MAVlink typically results in a laggy response caused by telemetry system lag and/or bandwidth limitations (these messages normally share bandwidth with other MAVLink messages passed between GCS and the vehicle)
+
+The Pilot's RC inputs can be monitored by the GCS or companion computer via the `RC_CHANNELS <https://mavlink.io/en/messages/common.html#RC_CHANNELS>`__ message
+
+RC_CHANNELS_OVERRIDE
+--------------------
+
+The `RC_CHANNELS_OVERRIDE <https://mavlink.io/en/messages/common.html#RC_CHANNELS_OVERRIDE>`__ message allows setting the PWM equivalent value for each channel.
+This is normally a value between 1000 and 2000 but ArduPilot's RC input library uses the RCx_MIN, RCx_MAX and RCx_TRIM parameters to scale the input.  Note though that these parameters are normally set by the user during the RC Calibration stage.
+If both a regular transmitter (e.g. Futaba, Spektrum, etc) which sends actualy PWM values and a MAVLink enabled RC input system (sending messages via one of the message listed on this page) then care should be taken to ensure the RCx_MIN/MAX ranges are consistent between the two systems
+
+Becareful of inadvertantly changing the flight mode.  By default either channel 5 or channel 8 (depending upon the vehicle type) is used to set the flight mode but this can be disabled by setting :ref:`FLTMODE_CH <copter:FLTMODE_CH>` or :ref:`MODE_CH <rover:MODE_CH>` to 0
```

Comment:
```suggestion
Be careful not to change the flight mode inadvertently.  By default, either channel 5 or channel 8 (depending upon the vehicle type) is used to set the flight mode but this can be disabled by setting :ref:`FLTMODE_CH <copter:FLTMODE_CH>` or :ref:`MODE_CH <rover:MODE_CH>` to 0
```

Reply:
I think override timeout should be mentioned in this section.....not just at the very end

Reply:
ok added